addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/booster/GolangBooster.java

```diff
@@ -0,0 +1,264 @@
+        boolean pushFile = false;
+        int z = NumberUtils.INTEGER_ZERO;
+        // If the file is a .go file, look for where the import starts
+        // to replace the git organization and set the index
+        // for the next loop.
+        if (isGoFile) {
+            int i;
+            for (i = NumberUtils.INTEGER_ZERO; i < content.size(); i++) {
+                String line = content.get(i);
+                if (line.contains(START_IMPORT)) {
+                    break;
+                }
+            }
+
+            z = i + NumberUtils.INTEGER_ONE;
+        }
+
+        // If we are processing imports in a .go file, break out of this
+        // loop as soon as we reach the end of imports. Otherwise, process
+        // the entire file searching to replace the git organization.
+        for (int i = z; i < content.size(); i++) {
+            String line = content.get(i);
+            if (isGoFile && line.contains(END_IMPORT)) {
+                break;
+            }
+
+            // If at any point the git organization was replaced the file
+            // must be pushed.
+            if (replaceContent(line, content, i, getGitOrganization(boosterData), this.gitUser)) {
+                pushFile = true;
+            }
+
+            if (replaceContent(content.get(i), content, i, getProjectName(boosterData), this.projectName)) {
+                pushFile = true;
+            }
+        }
+
+        return pushFile;
+    }
+
+    /**
+     * Extracts the boosters git organization.
+     *
+     * @param boosterData
+     *            The booster data to extract the git organization from.
+     * @return The git organization to replace while templatizing the booster.
+     */
+    private String getGitOrganization(Map<String, Object> boosterData) {
+        if (this.gitOrg != null) {
+            return this.gitOrg;
+        }
+        this.gitOrg = parseBoosterData(boosterData, NumberUtils.INTEGER_ONE);
+        return this.gitOrg;
+    }
+
+    /**
+     * Extracts the boosters project name in osio.
+     *
+     * @param boosterData
+     *            The booster data to extract the osio project name from.
+     * @return The osio project name to replace while templatizing the booster.
+     */
+    private String getProjectName(Map<String, Object> boosterData) {
+        if (this.osioProjectName != null) {
+            return this.osioProjectName;
+        }
+        this.osioProjectName = parseBoosterData(boosterData, NumberUtils.INTEGER_TWO);
+        return this.osioProjectName;
+    }
+
+    /**
+     * Extracts the boosters project name in osio.
+     *
+     * @param boosterData
+     *            The booster data to extract the osio project name from.
+     * @param segmentNum
+     *            The segment to return.
+     * @return The booster url segment to return.
+     */
+    @SuppressWarnings("unchecked")
+    private String parseBoosterData(Map<String, Object> boosterData, int segmentNum) {
```

Comment:
You don't need this method, you can call `getGitRepo()` from the RhoarBooster object inside the Projectile

Reply:
Ohh okay! That makes sense. Will change this!